mftf/2.3/configuration.md

```diff
@@ -56,21 +56,29 @@ Example:
 MAGENTO_ADMIN_PASSWORD=1234reTyt%$7
 ```
 
-## Advanced Configuration
+## Advanced configuration
 
-In most cases, these values are not required.
+The advanced configuration is optional.
 Sensible defaults are in place.
 But in case you do need to do some configuration, they are shown here for your reference.
 
-### SELENIUM
+### DEFAULT_TIMEZONE
 
-The `SELENIUM_*` values form the URL of a custom Selenium server for running testing.
+The [timezone][] used by a `generateDate` action that do not have a particular `timezone` specified.
```

Comment:
Change to ... that does not specify a time zone.  [or a `timezone` value]

Reply:
Will change to:
Sets a default value for the `timezone` attribute of a [`generateDate` action].
This value is applied when a test step does not specify a time zone.
For the complete list of available time zones, refer to [List of Supported Timezones].